src/components/reservation/TabCard.tsx

```diff
@@ -0,0 +1,68 @@
+import styled, { css } from 'styled-components';
+
+interface TabCardProps {
+	title: string;
+	totalPrice: string;
+	averageTime: string;
+	variant: 'cheapest' | 'shortest';
+}
+
+const TabCard = ({ title, totalPrice, averageTime, variant }: TabCardProps) => {
+	return (
+		<TabCardContainer variant={variant}>
+			<Title>{title}</Title>
+			<Price>₩ {totalPrice}</Price>
+			<Time>{averageTime} (평균)</Time>
+		</TabCardContainer>
+	);
+};
+
+export default TabCard;
+
+const TabCardContainer = styled.div<{ variant: 'cheapest' | 'shortest' }>`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 18rem;
+	height: 8.4rem;
+	padding: 1.1rem 5rem;
+	flex-direction: column;
+
+	// 추천순일 때 적용될 css
+	${({ variant }) =>
+		variant === 'cheapest' &&
+		css`
+			background-color: ${({ theme }) => theme.colors.darksky};
+			border-radius: 1rem 0rem 0rem 1rem;
+			${Title}, ${Price}, ${Time} {
+				color: ${({ theme }) => theme.colors.white}; /* cheapest일 때 텍스트 색상 */
+			}
+		`}
+
+	// 최단시간일 때 적용될 css
+	${({ variant }) =>
+		variant === 'shortest' &&
+		css`
+			background-color: ${({ theme }) => theme.colors.white};
+			border-radius: 0rem 1rem 1rem 0rem;
+		`}
+`;
```

Comment:
```suggestion
const TabCardContainer = styled.div<{ variant: 'cheapest' | 'shortest' }>`
   display: flex;
   align-items: center;
   justify-content: center;
   width: 18rem;
   height: 8.4rem;
   padding: 1.1rem 5rem;
   flex-direction: column;
   border-radius: ${({ variant }) =>
      variant === 'cheapest' ? '1rem 0rem 0rem 1rem' : '0rem 1rem 1rem 0rem'};
   background-color: ${({ theme, variant }) =>
      variant === 'cheapest' ? theme.colors.darksky : theme.colors.white};

   ${Title}, ${Price}, ${Time} {
      color: ${({ theme, variant }) =>
         variant === 'cheapest' ? theme.colors.white : theme.colors.black};
   }
`;
```
`variant`에 따른 스타일링이 중복되어 가독성이 많이 떨어지는 느낌이네요. 다음과 같이 기본 스타일을 추출하여 사용하면 가독성도 좋아지고 관리하기 편할 것 같아요.